user/snort/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "snort"
+pkgver = "3.9.1.0"
+pkgrel = 0
+build_style = "cmake"
+# "-DCMAKE_POLICY_VERSION_MINIMUM=3.5",
+configure_args = [
+    "-DDNET_INCLUDE_DIR=/usr/include/dnet",
+    "-DENABLE_LARGE_PCAP=ON",
+    "-DENABLE_SHELL=ON",
+]
+hostmakedepends = ["cmake", "flex", "ninja", "pkgconf"]
+makedepends = [
+    "daq-devel-static",
+    "hwloc-devel",
+    "libdnet-devel",
+    "libpcap-devel",
+    "libtirpc-devel",
+    "luajit-devel",
+    "numactl-devel",
+    "openssl3-devel",
+    "pcre2-devel",
+    "util-linux-uuid-devel",
+    "xz-devel",
+]
+depends = ["bash"]
```

Comment:
where is the bash dependency? i don't see it

Reply:
There is a bash script installed at `/usr/bin/appid_detector_builder.sh`

Reply:
hm, that seems like an auxiliary thing? i probably wouldn't add a hard bash dep just for that

Reply:
It's a useful tool for anyone running snort.  If you say we should remove, I will remove it but my vote is against it

Reply:
I can split it on a separate package that won't be installed by default